### ISSUE TITLE
Puzzle suggestion: Make the synchroscopes a puzzle

### ISSUE THREAD

**Issue by sebastiannielsen** [loiste-interactive/infra-issues#519]
A suggestion is to make the synchroscopes a puzzle, by having that button press will not automatically synchronize them, instead, the synchrocopes will land a specific distance counted from the button press, so you would have to time the button press to land the needle on the "In sync" position. And if you fail, you will have to push the button again to get then rotating again, and then redo the timing.

**Comment by Oskutin** [comment]
There's actually one:
https://www.youtube.com/watch?v=IW-MrsvKuJ0

**Comment by sebastiannielsen** [comment]
Yeah. But I meant a little more difficult version of what you show in the video, where theres is only one button that starts or stops the synchroscope, and then you have to time it right.

And this more difficult version can be used for the dam, since the puzzle in the dam is not required to solve, its just a "repairable spot".

While the puzzle in your video can be kept as is for that location.

A even more cool version would be that the synchroscope is tied to the sluice door control, so when the sluice door is set at 3000 RPM, the synchroscope will be static, when it moving >3000 RPM it will run forwards (lead) but more faster as more RPM over 3000, and when its <3000 RPM it will run backwards (lag) and more slower as lower RPM. Thus you would actually have to time the button press on the sluice door control to both land on 3000 RPM *AND* on "in sync", while the puzzle actually become very realistic.

(This would mean the single button on the synchroscope is removed, and possible the synchroscope is moved to the sluice panel above the RPM indicator).
That would make a really difficult and entertaining puzzle!

**Comment by CorruptComputer** [comment]
@sebastiannielsen I really like that idea, it would make that area more difficult, and you could set it up so that the player cannot progress without doing it. Like the room under the sawmill that you need to turn the power on.

**Comment by sebastiannielsen** [comment]
CorruptComputer: Naah, I really like the idea of the possible to fail the puzzle and get the major shortcut in the switchyard if the generators are not synced to the grid, but then you couldn't have it a mandatory puzzle. But the puzzle in the room in the YT video could be mandatory.

**Comment by Oskutin** [comment]
It is there.
